modules/auxiliary/scanner/http/meteocontrol_weblog_extractadmin.rb

```diff
@@ -0,0 +1,133 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'        => 'Meteocontrol WEBlog Password Extractor',
+      'Description' => %{
+          This module exploits an authentication bypass vulnerability in Meteocontrol WEBLog (all models). This vulnerability allows extracting Administrator password for the device management portal.
+      },
+      'References'  =>
+        [
+             [ 'URL', 'https://ics-cert.us-cert.gov/advisories/ICSA-16-133-01' ],
+             [ 'CVE', '2016-2296' ],
+             [ 'CVE', '2016-2298' ]
+        ],
```

Comment:
Thanks @h00die. I don't intend to add EDB ID reference in this module. I wrote it then just for quick check run. This PR is an updated, stable version.